PULL_REQUEST_TEMPLATE

```diff
@@ -1,24 +1,32 @@
 Jira: [JIRA-TOKEN](https://ticketevolution.atlassian.net/browse/JIRA-TOKEN)
 
+> [!IMPORTANT]
+> Toggle to SQUASH AND MERGE when merging PRs to `staging` branch
+
 ## Why / What
+Why: These changes are for...
```

Comment:
The reason we never had WHY here before is that it ends up duplicating much of what is in the Jira ticket. We have the link at the top of the PR request for this reason. The more interesting questions are what, and how.